Qadir__Work/HTML/Assignment6/style.css

```diff
@@ -0,0 +1,75 @@
+h1 {
+    font-size: x-large;
+}
+
+h1,
```

Comment:
not a prefect way Qadir ji

Reply:
Mein Apne way se set kiya hon font size as a image ko dehk ke.